docs/source/guides/contribution_guide.rst

```diff
@@ -5,6 +5,53 @@ Contributing
     :target: https://cdn-images-1.medium.com/max/800/1*J31AEMsTP6o_E5QOohn0Hw.png
 
 
+Acquiring the Codebase
+----------------------
+
+.. _`NuCypher GitHub`: https://github.com/nucypher/nucypher
+
+In order to contribute new code or documentation changes, you will need a local copy
+of the source code which is located on the `NuCypher GitHub`_.
+
+.. note::
+
+   NuCypher uses ``git`` for version control; Be sure you have it installed.
+
+Here is the recommended procedure for acquiring the code in preparation for
+contributing proposed changes:
+
+
+1. Use GitHub to Fork the `nucypher/nucypher` repository
```

Comment:
```suggestion
1. Use GitHub to Fork the ``nucypher/nucypher`` repository
```

Reply:
I actually intend for these to be rendered Italic as to not be confused with code.

Reply:
ok 👍

Reply:
:heavy_check_mark: